atomicapp/nulecule/base.py

```diff
@@ -100,6 +101,7 @@ def unpack(cls, image, dest, config=None, namespace=GLOBAL_CONF,
             docker_handler = DockerHandler(dryrun=dryrun)
             docker_handler.pull(image)
             docker_handler.extract(image, APP_ENT_PATH, dest, update)
+            Display().info("All dependencies installed successfully.", "cockpit")
```

Comment:
@dustymabe 
see @classmethod at the beginning of the function. We can define it again however within it though if you'd prefer.